### PR TITLE
Fix shortcut for single element

### PR DESCRIPTION
**Pull request by Dominator008**

### CODE REVIEW

File: contracts/optimistic-ethereum/libraries/utils/Lib_MerkleTree.sol [hunk 1]
```diff
@@ -34,7 +34,7 @@ library Lib_MerkleTree {
             "Lib_MerkleTree: Must provide at least one leaf hash."
         );
 
-        if (_elements.length == 0) {
+        if (_elements.length == 1) {
             return _elements[0];
```
Comment (maurelian) [inline contracts/optimistic-ethereum/libraries/utils/Lib_MerkleTree.sol:38]: Good catch. It looks like this line is not even reachable prior to the change.

**Review by maurelian — APPROVED** [review]
This looks like a good catch to me!

**Comment by snario** [comment]
Dominator!!!!

**Comment by Dominator008** [comment]
Hmm, I think the actual test is passing, right?

**Comment by snario** [comment]
> Hmm, I think the actual test is passing, right?

There was an issue that @karlfloersch just resolved causing the CI to incorrectly error. Tests are ✅ now!

**Pull request merged**